ReleaseHistory.md

```diff
@@ -1,4 +1,26 @@
-# Microsoft.Security.Utilities Release History
+# Microsoft.Security.Utilities.Core Release History
+- NR  => new rule
+- PRF => performance work
+- FCR => fingerprint change or refactor
+- RRR => rule rename or refactor
+- FPC => regex candidate reduction
+- FNC => regex candidate increase
+- FPS => FP reduction in static analysis
+- FNS => false negative reduction in static analysis
+- FPD => FP reduction in dynamic phase
+- FND => False negative reduction in dynamic phase
+- UER => eliminate unhandled exceptions in rules
+- UEE => eliminate unhandled exceptions in engine
+- DEP => upgrade dependency versions
+- NEW => new feature 
+
+UNRELEASED
```

Comment:
@michaelcfanning We have `docs/ReleaseNotes.md`. Is the plan to abandon that and maintain updates in `ReleaseHistory`?